nmrfx-analyst/src/main/java/org/nmrfx/analyst/dataops/VecCorrelation.java

```diff
@@ -603,8 +608,7 @@ public static PositionValue fftCorr(double[] vecR, double[] vecS, int maxShift)
 //            System.out.println("shift " + shift + " n " + n + " clen " + corr_rs_power.length + " " + maxShift);
             pValue = new PositionValue(shift, max_corr);
         } catch (Exception e) {
-            System.out.println("Exception " + e.toString());
-            e.printStackTrace();
+            log.error(e.getMessage(), e);
             System.exit(-1);
```

Comment:
Used log.error instead of log.warn when System.exit(-1) is called immediatly after